### PR TITLE
feat(interview-prep): Porting Rosetta problems

### PR DESCRIPTION
**Pull request by NitronR**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `dev` branch.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/challenge-tests`)
- [ ] All new and existing tests pass the command `npm test`.
- [ ] Use `npm run commit` to generate a conventional commit message.
    Learn more here: <https://conventionalcommits.org/#why-use-conventional-commits>
- [ ] The changes were done locally on your machine and NOT GitHub web interface.
    If they were done on the web interface you have ensured that you are creating conventional commit messages.

#### Checklist:

- [ ] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

### CODE REVIEW

File: challenges/08-coding-interview-prep/rosetta-code.json [hunk 1]
```diff
@@ -2621,6 +2851,206 @@
...
+          ],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Doubly-linked list/Definition",
+      "description": [
+        "Write a function that takes an array as parameter. This array contains queries that specify operations to be performed on a doubly linked list. The operations are in the form [type,args...]. The type can be head, tail or after.",
```
Comment (scissorsneedfoodtoo) [inline challenges/08-coding-interview-prep/rosetta-code.json]: "Write a function that takes an array as a parameter. This array contains queries that specify operations to be performed on a doubly linked list. The operations are in the form of <code>[ type, args... ]</code>. The type can be <code>head</code>, <code>tail</code> or <code>after</code>."

File: challenges/08-coding-interview-prep/rosetta-code.json [hunk 2]
```diff
@@ -2621,6 +2851,206 @@
...
+        "For head, insert the second element of the operation to the beginning of the list. For tail, insert at end of list. For after, the second and third element of the operation will be index and value. The value should be inserted after the given index.",
```
Comment (scissorsneedfoodtoo) [inline challenges/08-coding-interview-prep/rosetta-code.json]: `"If the type is <code>head</code>, insert the second element of the operation to the beginning of the list. If the type is <code>tail</code>, insert at end of list. If it's <code>after</code>, the second and third elements of the operation will be an index and a value. The value should be inserted after the given index."`

**Comment by NitronR** [comment]
@scissorsneedfoodtoo Changes done.

**Comment by scissorsneedfoodtoo** [comment]
Alright! Looks great. As always, thank you for all of your hard work bringing these challenges over.

**Pull request merged**